cohere/moderate.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+from cohere.response import CohereObject
+
+
+class Moderation(CohereObject):
+
+    def __init__(self, benign: float, profanity: float, hate_speech: float, violence: float, self_harm: float,
+                 sexual: float, sexual_non_consensual: float, spam: float, information_hazard: float) -> None:
+        self.benign = benign
+        self.profanity = profanity
+        self.hate_speech = hate_speech
+        self.violence = violence
+        self.self_harm = self_harm
+        self.sexual = sexual
+        self.sexual_non_consensual = sexual_non_consensual
+        self.spam = spam
+        self.information_hazard = information_hazard
+
+
+class Moderations(CohereObject):
```

Comment:
Do we need this wrapper? Could we just return a list of `ModerationResult`?

Reply:
I think in the other APIs we use this wrapper so that we can attach an ID to it. So i guess either attach the response ID, or we can get rid of the wrapper.